src/components/Menu.js

```diff
@@ -34,6 +111,49 @@ const renderMenu = (menuItem) => {
             `}
           />
 
+          <${Expandable}
+            toggle=${'Legend'}
+            detail=${html`
+              <p>The map shows two parameters for territories:</p>
+              <ol>
+              <li>If the country is under any type of lockdown</li>
+              <li>If there are covid cases in the country</li>
+              </ol>
+            <table class=${styles}>
+<tr>
+<th></th>
+<th>Lockdown</th>
+<th>COVID-19</th>
+</tr>
+<tr>
+<td><div class="blue circle"/></td>
```

Comment:
This has no semantic value for users with a screen reader, can you look into this? Let me know if you need help :)

Reply:
I dont understand for this part. What do you mean by no semantic value for users with a screen reader? What should I do here?

Reply:
Some users with a vision disablity may use a screen reader to use the application. Currently, this element has no value to relay any information to the screen reader (meaning that the screen reader will just ignore this element, and people with a vision disability will not have access to this information). You can solve this by adding a `aria-label="blue"` to these divs